src/main/java/com/spotinst/sdkjava/model/SpotinstElastigroupRepo.java

```diff
@@ -836,4 +836,204 @@ public RepoGenericResponse<List<GetInstanceTypesResponse>> getSuggestedInstanceT
 
         return retVal;
     }
+
+    @Override
+    public RepoGenericResponse<ElastigroupEcsClusterRollResponse> initiateClusterRollInEGWithECS(ElastigroupEcsInitiateRoll rollRequest, String groupId, String authToken, String account) {
```

Comment:
Can we move this repo class to repo folder.

Reply:
showing so many conflicts during refactor, will do it later